### PR TITLE
fixes for haxe

### PR DESCRIPTION
**Pull request by nadako**

### CODE REVIEW

File: haxe.html.markdown [hunk 1]
```diff
@@ -11,7 +12,7 @@ Haxe author).  Note that this guide is for Haxe version 3.  Some of the guide
 may be applicable to older versions, but it is recommended to use other
 references.
 
-```csharp
+```haxe
```
Comment (levibostian) [inline haxe.html.markdown]: Sorry, but haxe [is not supported](https://github.com/adambard/rouge/tree/master/lib/rouge/lexers) in the website. Will have to use csharp as it is the best alternative.

File: haxe.html.markdown [hunk 2]
```diff
@@ -34,16 +35,20 @@ references.
 /*
    This is your first actual haxe code coming up, it's declaring an empty
    package.  A package isn't necessary, but it's useful if you want to create a
-   namespace for your code (e.g. org.module.ClassName).
+   namespace for your code (e.g. org.yourapp.ClassName).
+
+   Omitting package declaration is the same as declaring empty package.
```
Comment (levibostian) [inline haxe.html.markdown]: `declaring empty package` => `declaring an empty package`

File: haxe.html.markdown [hunk 3]
```diff
@@ -383,11 +395,7 @@ class LearnHaxe3{
         */
 
         // if statements
-        var k = if (true){
-            10;
-        } else {
-            20;
-        }
+        var k = if (true) 10 else 20;
```
Comment (levibostian) [inline haxe.html.markdown]: Is using one line for conditionals in the style guidelines?

File: haxe.html.markdown [hunk 4]
```diff
@@ -628,6 +636,7 @@ enum ComplexEnum{
     ComplexEnumEnum(c:ComplexEnum);
 }
 // Note: The enum above can include *other* enums as well, including itself!
+// Note: This is what called *Algebraic data type* in some other languages.
```
Comment (levibostian) [inline haxe.html.markdown]: `what` => `whats`

**Comment by levibostian** [comment]
Good to go after checking over those quick notes I made. Thanks!

File: haxe.html.markdown (hunk 4 above)
Comment (vendethiel) [inline haxe.html.markdown]: (`what's`, even)

File: haxe.html.markdown (hunk 3 above)
Comment (nadako) [inline haxe.html.markdown]: It's somehow not obvious that one can use more lightweight syntax without curly braces. I think this guide also need to mention that even functions don't require `{}`, but maybe later.

**Comment by nadako** [comment]
Done.

**Comment by levibostian** [comment]
Thanks!

**Pull request merged**